Description: qw() in list context is an error with newer perls
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=85966
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=85966
Bug-Debian: http://bugs.debian.org/711394
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-16

--- a/lib/User/Simple/Admin.pm
+++ b/lib/User/Simple/Admin.pm
@@ -339,7 +339,7 @@
 	    # sensitive. Gah, we work around that to provide the much more 
 	    # common lowercase fields... This might still have some problems
 	    # attached, please tell me if it breaks for you.
-	    for my $case qw(id ID Id iD) {
+	    for my $case (qw(id ID Id iD)) {
 		if (exists $row->{$case}) {
 		    $id = $row->{$case};
 		    last;
